### PR TITLE
updates email flow based on status

### LINKED ISSUE #34: "Thank you for submitting a session..." email should only go out once in Submitted status
The "Thank you for submitting a session..." email should only go out once the request is in a "Submitted" status. Right now, the email is triggering as soon as the user starts the session submission process.

We should also have an email sent to the user after X amount of time has passed with the session in "Draft" status - I would suggest 24 hours.

### PR DESCRIPTION
**Pull request by csell5**
Closes #34

### CODE REVIEW

File: src/graphql/resolvers/mutations/session.js [hunk 1]
```diff
@@ -23,24 +23,43 @@ export const fieldResolvers = {
...
+        orginalSession.status === 'DRAFT' &&
+        updatedSession.status === 'SUBMITTED'
+      ) {
+        userEvents.emit('newSessionCreated', {
+          user: userResults,
+          session: updatedSession,
+        });
+      }
+
+      if (updatedSession.status === 'ACCEPTED') {
```
Comment (brettski) [inline src/graphql/resolvers/mutations/session.js]: If a session is updated after it is ACCEPTED, will another accepted email go out?

File: src/graphql/typeDefs/dataTypes/enums/status.graphql [hunk 2]
```diff
@@ -6,4 +6,5 @@ enum Status {
   WITHDREW
   WAIT_LIST
   CANCELLED
+  SCHEDULED
```
Comment (brettski) [inline src/graphql/typeDefs/dataTypes/enums/status.graphql]: Good thought. Do we have a flag, or condition for sessions which cannot be rescheduled?

Comment (csell5) [reply src/graphql/typeDefs/dataTypes/enums/status.graphql]: good thought.. nope..  I was just starting to think about things like your above statement and how we might want to work this a bit differently to support a richer set of scenarios in the future.

File: src/graphql/resolvers/mutations/session.js (hunk 1 above)
Comment (csell5) [reply src/graphql/resolvers/mutations/session.js]: right now yes. but we don't say what was changed, etc.. which got me thinking about scheduling. maybe we pull it for now until we have something better. it's kinda like telling the user, yea our db save worked.. meh..

**Pull request merged**